testmocks/src/main/java/org/apache/zookeeper/TestZKServer.java

```diff
@@ -16,32 +16,27 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.pulsar.metadata;
+package org.apache.zookeeper;
```

Comment:
Why did you move this class?

Reply:
the new added testcase dependence on `TestZKServer`